.github/workflows/flake8.yaml

```diff
@@ -0,0 +1,23 @@
+name: flake8 lint
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: "0 8 * * *"
```

Comment:
**question (blocking):** is there a benefit to running this on an interval, instead of just on PR/merge/tag?

Reply:
Agreed, for flake8, probably only worth running on PR/merge etc. Removed.